mantidproject/www
# Initial setup of the site and implement index page

**Description of Work**
This PR sets up the new website using sphinx and the bootstrap theme. It also implements an initial version of the index page and custom styling.

**To Test**
1. Run `make html`
2. Open `build > html > index.html` in a web browser.
3. Check the index page works and looks as expected.

Fixes #2
